src/protocol/protocol.h

```diff
@@ -78,4 +78,7 @@ PROTODEF(PROTOCOL_PXX, R9M,  NULL, PXXOUT_Cmds, "PXX")
 PROTODEF(PROTOCOL_TESTSER, TX_MODULE_LAST,  NULL, TESTSER_Cmds, "TESTSER")
 PROTODEF(PROTOCOL_TESTRF, TX_MODULE_LAST,  NULL, TESTRF_Cmds, "TESTRF")
 #endif
+#if HAS_SCANNER
```

Comment:
SUPPORT_SCANNER.

HAS_ is used to choose hardware. SUPPORT_ is used to define an optional software feature. But what's the reason we want to this feature be optional?

Reply:
Agree on SUPPORT_. Making this optional until we know Rom and Ram impact on 7e

Reply:
Also, scanner doesn't work for text gui.